### ISSUE TITLE
Softplus with parameters

### ISSUE THREAD

**Issue by nishnik** [tensorflow/tensorflow#11892]
Should I go on implementing the parametric version of Softplus?

Current implementation: [softplus_impl.py#L35](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/distributions/python/ops/bijectors/softplus_impl.py)

Reference to parametric version: [PyTorch docs](http://pytorch.org/docs/master/nn.html?highlight=softplus#torch.nn.Softplus)

**Comment by shivaniag** [comment]
@ jvdillon could you please comment on this.

**Comment by jvdillon** [comment]
Sorry for just noticing this issue.  Im happy to make this change ASAP.

**Comment by jvdillon** [comment]
So we do have the `hinge_softness` parameter. Do you need the `threshold` parameter as well? (I question if `threshold` is useful since the function is anyway almost linear...)

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.
